src/App.tsx

```diff
@@ -1,35 +1,56 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState } from "react";
+import Stopwatch from "./components/Stopwatch";
+import Add from "./assets/add.svg";
+import Delete from "./assets/delete.svg";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const prevStopwatches = JSON.parse(localStorage.getItem("ids") || "[]");
+  const prevCurrentId = parseInt(localStorage.getItem("currentId") || "0");
 
+  const [stopwatches, setStopwatches] =
+    useState<Array<number>>(prevStopwatches);
+  const [currentId, setCurrentId] = useState(prevCurrentId);
+
+  useEffect(() => {
+    localStorage.setItem("ids", JSON.stringify(stopwatches));
+    localStorage.setItem("currentId", currentId.toString());
+  }, [stopwatches, currentId]);
+
+  function handleAdd() {
+    setStopwatches((prev) => [...prev, currentId]);
+    setCurrentId((prev) => prev + 1);
+  }
+
+  function handleDelete(idToDelete: number) {
+    setStopwatches((prev) => prev.filter((id) => id !== idToDelete));
```

Comment:
you don't need to use localstorage for this task in general, only case is to save stopwatches when user leaves the page. This idea is good, but if you want to implement it, it would be nice to save, how much time is left + moment when it was saved, so when user returns to page, stopwatches continue not from the moment page left, but taking an account how much time left in general. Possibly it could lead to some problems with device time, but I believe it's not the problem you need to solve in current hometask